File: AppApi/Helpers/UserParams.cs
namespace AppApi.Helpers
{
    public class UserParams : PaginationParams
    {
        public string Username { get; set; }
        public string Type { get; set; }
        public string Status { get; set; }
        public string State { get; set; }
        public int MinMonth { get; set; } = 1;
        public int MinYear { get; set; } = 2022;
        public int MaxMonth { get; set; } = DateTime.Now.Month;
        public int MaxYear { get; set; } = DateTime.Now.Year;
        public string OrderBy { get; set; } = "dateLatest";

    }
}

File: client/src/app/models/userParams.ts
export class UserParams {
    pageSize = 8;
    pageNumber = 1;
    type = '';
    status = '';
    minMonth = 1;
    maxMonth = (new Date()).getMonth() + 1
    minYear = 2022
    maxYear = (new Date()).getFullYear()
    orderBy = '';
    state = ''
}